Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs

```diff
@@ -67,41 +80,40 @@ public void E2E_OutputFolderStructure()
         public void E2E_MissingProjectGuid()
         {
             // Projects with missing guids should have a warning emitted. The project info
-            // should not be generated.
+            // should still be generated.
```

Comment:
The previous comment was incorrect - the test was checking that the projectInfo.xml file was being created.